draft-xu-risav.md

```diff
@@ -125,22 +128,19 @@ Signature:
+## What RISAV Is
 
-3. IKE process. Before IPsec establishing, the SA must be reached an agreement. There are two ways to negotiate the SA. One is mutual config all the parameters and the other one is using IKE for dynamic negotiating these parameters. Currently used is IKE version 2 (IKEv2, for short using IKE below). Typically, IKE would produce IKE SA in the IKE_SA_INIT exchange and IPsec SA in the IKE_AUTH exchange. This will be done in an ACS.
+RISAV is a cryptographically-based inter-AS source address validation approach that guarantees security benefits at partial deployment. It aims to provide the IP datagram with a valid source address, with the capability of anti-spoofing, anti-replay, light-weight and efficient, and incremental deployment incentives. As a result, RISAV adds a tag to a packet at the source AS Border Router (ASBR) proving that the packet is with a valid source address, and it would verify and remove this tag at the destination ASBR. The tag will be encapsulated in the Integrity Check Value (ICV) field of IPsec AH/ESP.
 
-4. SA or tag delivery. When all negotiations are done, the IPsec is established. If the ACS is an ASBR actually, it would deliver SA to the other ASBR in this AS. Otherwise, it would deliver the tags generated by the state machine. This delivery will be processed in a secure channel just like using IPsec ESP.
+## How RISAV Works
 
-5. IPsec AH communication. It uses IPsec AH for authentication of the IP source address.  IPsec is often used in tunnel mode as the IPsec VPN. Here, It expands the gateway to the AS border router (ASBR). When two ends x and y in AS X and Y respectively are communicating, the packet from x arriving at its ASBR RA would use the established IPsec channel for adding the representative tag. After the packet arrives at ASBR RB of AS Y, it would be inspected by comparing the consistency of the tag. The tag will be encapsulated in the ICV field in AH.
+RISAV uses static-static ECDH as an alternative of IKE {{RFC8247}} to negotiate the security association (SA) used in IPsec AH/ESP communications. Otherwise, it MUST follow strictly the standard IKE process to negotiate IKE SA and IPsec SA. RPKI obtains the binding relationship between AS numbers and IP prefixes, and it will synchronize the public key generated by ECDH and other messages including the contact IP to all peers. And the original IPsec AH/ESP header format is used in communication to carry this tag. The transport mode of IPsec AH is applied in general.
 
-A typical workflow of RISAV is shown in {{figure1}}.
+Before deploying RISAV, each AS sets a contact IP representative. When negotiating or consulting with one AS, the peer MUST first communicate with this contact IP. This contact IP should contain at least and at most two IPs: one is IPv4 and the other is IPv6. In short, these are referred to as contact IP below.
```

Comment:
I think we can simplify the requirement on the number of contact IPs.

```suggestion
Before deploying RISAV, each AS sets a contact IP representative. When negotiating or consulting with one AS, the peer MUST first communicate with this contact IP. The AS MUST publish exactly one contact IP for each supported address family (i.e. IPv4 and/or IPv6).
```

Reply:
Yes, there may be some engineering considerations that I miss, like hot standby.

Reply:
So sorry, it's my fault. I may wrongly commit & merge as not familiar with GitHub. I would add them when submitting the next PR as this PR has been closed.